modules/core/src/main/scala/Tags.scala

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2019 by Rob Norris
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package natchez
+
+object Tags {
+
+  // The software package, framework, library, or module that generated the associated Span. E.g., "grpc", "django", "JDBI".
+  def Component(c: String) = Field("component", c)
```

Comment:
What if instead of `Field` these just returned a `(String, TraceValue)` pair? That way you could mix and match known tags and arbitrary pairs, like

```scala
Trace[F].put(Component("foo"), "bar" -> 42, ...)
```